www/api/class/centreon_monitoring_externalcmd.class.php

```diff
@@ -34,8 +34,8 @@
  */
 
 require_once _CENTREON_PATH_ . "/www/class/centreonDB.class.php";
+require_once _CENTREON_PATH_ . 'www/class/centreonExternalCommand.class.php';
```

Comment:
prefer adding `/` in case of trailing slash is missing in `_CENTREON_PATH_`
```suggestion
require_once _CENTREON_PATH_ . '/www/class/centreonExternalCommand.class.php';
```